adaptive_shield/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "manifest_version": "2.0.0",
+    "app_id": "adaptive-shield",
+    "app_uuid": "",
+    "display_on_public_website": true,
+    "tile":
+    {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Track SaaS posture alerts",
+        "title": "Adaptive Shield",
+        "media":
+        [],
```

Comment:
We recommend adding a few images - these can be screenshots of your dashboard, or of your SaaS platform as well. Images can be added with the following syntax:

```
{
    "caption": "caption goes here",
    "image_url": "images/path-to-image.png",
    "media_type": "image"
},
```

Reply:
The images that you add here will be automatically displayed in a media carousel on the tile, no need to reference them in the readme.md file.